plugins/login-plugin/src/Components/Login/LoginTv/index.js

```diff
@@ -33,8 +33,8 @@ const styles = StyleSheet.create({
     marginRight: 168,
   },
   clientLogoView: {
-    height: 100,
-    width: 350,
+    height: 58,
+    width: 204,
```

Comment:
Why to change it? I think we should keep it as it is in the design https://scene.zeplin.io/project/5de6c3488f3ec0487deb2374/screen/5dee714208ccb013801ee138